include/LogicBlock/Z3Logic.hpp

```diff
@@ -94,11 +94,11 @@ namespace z3logic {
         void          internalReset() override;
 
     public:
-        Z3LogicOptimizer(z3::context& ctx, z3::optimize& optimizer,
-                         bool convertWhenAssert = true):
-            LogicBlockOptimizer(convertWhenAssert),
-            Z3Base(ctx),
-            optimizer(optimizer) {}
+        Z3LogicOptimizer(z3::context& context, z3::optimize& opt,
+                         bool convert = true):
+            LogicBlockOptimizer(convert),
+            Z3Base(context),
+            optimizer(opt) {}

```

Comment:
```suggestion
        Z3LogicOptimizer(z3::context& context, z3::optimize& optimizer,
                         bool convert = true):
            LogicBlockOptimizer(convert),
            Z3Base(context),
            optimizer(optimizer) {}

```

Reply:
That suggestion just introduces the warning again 🙃

Reply:
I'll leave it for now and refactor the class members sometime.